kong/init.lua

```diff
@@ -344,19 +345,19 @@ function Kong.init()
   sort_plugins_for_execution(config, db, loaded_plugins)
 
   local err
-  plugins_map_semaphore, err = semaphore.new()
+  plugins_map_semaphore, err = semaphore.new(1) -- 1 = treat this as a mutex
   if not plugins_map_semaphore then
     error("failed to create plugins map semaphore: " .. err)
   end
 
-  plugins_map_semaphore:post(1) -- one resource, treat this as a mutex
```

Comment:
Curious, why this change? Seems unrelated?

Reply:
@p0pr0ck5 nothing, just a small cleanup as I was touching code close to it. Not a big deal to include or remove the change.